src/store.h

```diff
@@ -74,6 +74,10 @@ class MetadataStore {
 
   MetadataStore(const Configuration& config);
 
+  // Returns a copy of the mapping from monitored resources to metadata
```

Comment:
`from a monitored resource to` — otherwise one is plural and one is singular.
Also `the metadata`.
So:
```cpp
  // Returns a copy of the mapping from a monitored resource to the
  // metadata associated with that resource.
```

Reply:
Done.